src/ApiService/ApiService/onefuzzlib/Extension.cs

```diff
@@ -136,13 +136,6 @@ private class Settings {
     }
 
     public static VMExtensionWrapper AzMonExtension(AzureLocation region, AzureMonitorExtensionConfig azureMonitor) {
```

Comment:
you can test this out without submitting PR can't you ?

Reply:
or you validated and now this is a good change ?

Reply:
You're right. I don't need to submit PR just yet.